Gruntfile.js

```diff
@@ -37,12 +43,41 @@ module.exports = function (grunt) {
 					dest: 'dist/'
 				}]
 			}
+		},
+		postcss: {
+			options: {
+				map: true,
+				processors: [
+					require('postcss-simple-vars')({
+						variables: require('./src/themes/structural/common')
+					}),
+					require('postcss-modules')({
+						getJSON: function(cssFileName, json) {
+							var filename = path.basename(cssFileName, '.css');
+							fs.writeFileSync(
+								'src/themes/structural/modules/' + filename + '.ts',
+								'export default ' + JSON.stringify(json) + ';'
```

Comment:
Switched to string literals so concatenation is no longer necessary and added a new line to each generated file.